src/components/progressiveImage/progressiveImage.api.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ProgressiveImage",
+  "category": "animationsAndGestures",
+  "description": "Image component that loads first a small thumbnail of the images, and fades-in the full-sized image with animation once it's loaded",
+  "extends": ["Animated.Image"],
```

Comment:
What's interesting in extends is to let the user understand what other props are supported. so I think `Image` is enough in this case. WDYT?

Reply:
mmm, I see we're using our `AnimatedImage` and not actually `Animated.Image` directly, so there are other props, should I change it to `AnimatedImage` or `Image`?

Reply:
If AnimatedImage is our component then yes let's change to AnimatedImage, otherwise Image is enough

Reply:
Done